src/test/java/org/broadinstitute/hellbender/utils/TranscriptUtilsUnitTest.java

```diff
@@ -0,0 +1,246 @@
+package org.broadinstitute.hellbender.utils;
+
+import htsjdk.samtools.util.Locatable;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.engine.ReferenceDataSource;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.testutils.FuncotatorReferenceTestUtils;
+import org.broadinstitute.hellbender.tools.funcotator.dataSources.gencode.GencodeFuncotationFactory;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TranscriptUtilsUnitTest extends GATKBaseTest {
+
+    //==================================================================================================================
+    // Private Static Members:
+
+    private static final ReferenceDataSource refDataSourceHg19Ch3;
+    private static final ReferenceDataSource refDataSourceB37;
+
+    private static final List<AutoCloseable> autoCloseableList = new ArrayList<>();
+    static {
+        refDataSourceHg19Ch3 = ReferenceDataSource.of( IOUtils.getPath(FuncotatorReferenceTestUtils.retrieveHg19Chr3Ref()) );
+        refDataSourceB37 = ReferenceDataSource.of( IOUtils.getPath(b37Reference) );
```

Comment:
How difficult would it be to check in a chunk of datasource that HAS these transcript files in it and just call the `get___fromReference()` methods and directly do the string comparison with the transcript file that is checked in from gencode directly? That seems like the strongest possible test and not too terribly difficult to write

Reply:
Good idea.  I think we should have this available for testing already.  I can add another test to do this.